<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountReg</class>
 <widget class="QDialog" name="AccountReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register Account</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="sw_register">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_server">
      <layout class="QVBoxLayout">
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gb_server">
         <property name="title">
          <string>Server</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="lb_jid">
            <property name="text">
             <string>Please enter the name of the server you wish to register with:</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="UpdatingComboBox" name="le_server" >
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="lb_example">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Example: capulet.com</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_connection">
         <property name="title">
          <string>Connection settings</string>
         </property>
         <layout class="QVBoxLayout">
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="ck_host">
            <property name="text">
             <string>Manually Specify Server Host/Port:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="lb_host">
              <property name="text">
               <string>Host:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_host"/>
            </item>
            <item>
             <widget class="QLabel" name="lb_port">
              <property name="text">
               <string>Port:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_port">
              <property name="minimumSize">
               <size>
                <width>56</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>56</width>
                <height>32767</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="lb_ssl">
              <property name="text">
               <string>Encrypt connection:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_ssl"/>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>151</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="lb_proxy">
              <property name="text">
               <string>Proxy:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_proxychooser">
              <property name="text">
               <string>proxychooser</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_fields"/>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line2">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="BusyWidget" name="busy" />
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_next">
       <property name="text">
        <string>&amp;Next</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>le_server</tabstop>
  <tabstop>ck_host</tabstop>
  <tabstop>le_host</tabstop>
  <tabstop>le_port</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
